src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -134,7 +134,7 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ proj, enter
     </div>
   );
 
-  const hasTaskPermissions =
+  const createTaskPermissions =
```

Comment:
why the name change here

Reply:
or I guess more so if we change the name here why don't we change the prop we're passing it into as well

Reply:
sure, i can separate create and edit permissions in every component